src/app.ts

```diff
@@ -189,14 +197,13 @@ export async function getApp(
 
   await app.register(jwtTokenPlugin, {
     skipList: new Set([
+      '/favicon.ico',
       '/login',
       '/access-token',
       '/refresh-token',
       '/documentation',
       '/documentation/json',
-      '/documentation/static/*',
-      '/documentation/static/index.html',
-      '/documentation/static/swagger-initializer.js',
+      '/documentation/@scalar/fastify-api-reference/js/browser.js',
```

Comment:
do we need to update e2e tests and readme accordingly?

Reply:
Do you have any particular places in mind?

AFAIK, docs are still at `/documentation`, but we need a js to render them.

Reply:
`You can access OpenAPI specification of your application, while it is running, by opening [SwaggerUI](http://localhost:3000/documentation)`

It's no longer SwaggerUI, but you are saying that the path is still the same?

See https://github.com/lokalise/node-service-template/blob/main/src/app.e2e.spec.ts, `/documentation/json` is also still valid?

Reply:
oh, this
yeah, I will update a link name. Path is still the same, this is a drop-in replacement. `/documentation/json` is also still valid